src/components/pipelines-details/__tests__/pipeline-step-utils.spec.ts

```diff
@@ -0,0 +1,240 @@
+import { ComputedStatus, PipelineRunKind } from '../../../types';
+import { sortPipelineRunsByDuration } from '../pipeline-step-utils';
+
+// Mock the pipeline-filter-reducer module
+jest.mock('../../utils/pipeline-filter-reducer', () => ({
+  pipelineRunStatus: jest.fn(),
+}));
+
+import { pipelineRunStatus } from '../../utils/pipeline-filter-reducer';
```

Comment:
nit: All imports should at one place.

```suggestion
import { pipelineRunStatus } from '../../utils/pipeline-filter-reducer';
import { sortPipelineRunsByDuration } from '../pipeline-step-utils';

// Mock the pipeline-filter-reducer module
jest.mock('../../utils/pipeline-filter-reducer', () => ({
  pipelineRunStatus: jest.fn(),
}));

```